### ISSUE TITLE
default SOLR query not searching text

### ISSUE THREAD

**Issue by jeff-at-h3** [ckan/ckan#4235]
### CKAN Version if known (or site URL)
2.7.2
Solr 7

### Please describe the expected behaviour
Using the search feature for datasets, providing text, eg. parks, returns datasets the have the work park (plus plurals) in the results.

### Please describe the actual behaviour
Our test env is using Solr 7 and our production env is using Solr 4.10, querying with 'park', there is no results with the search. 

### What steps can be taken to reproduce the issue? 
For the BCgov catalog, we haven't changed anything relating to the core serach functionality.

Querying the catalog with something like `/dataset?q=park` in our Production env, this works as expected, but in our upgraded envs, the same query returns no results.

However, specifying a field, eg. `text:park`, returns results as we expected.

In issue #3510, it seems like this is a similar problem?

### Questions
1. Has the default behaviour for `q` changed between Solr versions? @

**Comment by dkelsey** [comment]
We have an extension that was injecting ```{!lucene q.op=AND}``` into the query.
The search query that resulted from a query for `park` was ```{!lucene q.op=AND}park```.
This was failing to return results. 
I removed the _injection_ of ```{!lucene q.op=AND}``` and the issue appears to be resolved.
We'll be testing to assert that this **does** resolve the issue.

**Comment by smotornyuk** [comment]
Yes, i believe, that starting from v6.6 you need explicit definition of default query field. I'm using ```      <str name="df">text</str>
      <str name="q.op">AND</str>``` under `//requestHandler[@name=/select]/lst[@name="defaults]`(and some other changes) inside solrconfig.xml.
